server/api/api/endpoints/model_endpoints.py

```diff
@@ -349,7 +349,10 @@ async def get_model_endpoint_monitoring_metrics(
     )
 
     get_model_endpoint_metrics = mlrun.model_monitoring.get_store_object(
-        project=project
+        project=project,
+        secret_provider=server.api.crud.secrets.get_project_secret_provider(
+            project=project
+        ),
```

Comment:
I see this repeats in multiple places, how about a helper function?
E.g., in `server/api/crud/model_monitoring/model_endpoints.py` L143:
```py
        model_endpoint_store = mlrun.model_monitoring.get_store_object(
            project=model_endpoint.metadata.project,
            secret_provider=server.api.crud.secrets.get_project_secret_provider(
                project=model_endpoint.metadata.project
            ),
        )
```